src/app/modules/shifts/actions.ts

```diff
@@ -14,8 +14,9 @@ import {
 export const getShifts = shiftRequests.shiftMapRequest.actionCreator;
 export const createShift = shiftRequests.createShiftRequest.actionCreator;
 export const copyShiftsFromPrevWeek = shiftRequests.copyShiftsFromPrevWeek.actionCreator;
-export const editShift = shiftRequests.updateShiftRequest.actionCreator;
+export const editMultipleShifts = shiftRequests.updateSelectedShiftsRequest.actionCreator;
```

Comment:
similar comment to above.  I think this request could be called `updatedMultipleShiftsRequest`